cmd/call-endpoint/main.go

```diff
@@ -34,13 +36,26 @@ func main() {
 		return
 	}
 
-	var endpointDescription *core.EndpointDescription
+	if args.List {
+		listAPIEndpoints()
+
+		// Return to prevent further execution after listing endpoints.
```

Comment:
I don't think the comment (or space) are necessary here.
What you are doing is pretty clear.